src/sagemaker_container_support/environment.py

```diff
@@ -0,0 +1,607 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License'). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the 'license' file accompanying this file. This file is
+# distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import json
+import logging
+import multiprocessing
+import os
+import shlex
+import subprocess
+from typing import Any, Dict, List, Tuple, Union  # noqa: F401
+
+import boto3
+
+from six import PY2, reraise
+
+if PY2:
+    JSONDecodeError = None
+else:
+    from json.decoder import JSONDecodeError
+
+logging.basicConfig()
+logger = logging.getLogger(__name__)
+
+BASE_PATH_ENV = 'BASE_PATH'  # type: str
```

Comment:
remove type hints

Reply:
The specific example above is just a comment without important any library type or making any enforcement.

Let me know if you are of with these kind of comment. I am not importing typing anymore.